File: trail1/출석 부르기/calling-attendance.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {
    vector<string> arr = {"John", "Tom", "Paul"};
    int N;
    cin >> N;
    if (N<=3){
        cout << arr[N-1] << endl;
    }
    else {cout << "Vacancy" << endl;}
    return 0;
}

File: trail1/출석 부르기/calling-attendance.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] arr = {"John", "Tom", "Paul"};
        int N = sc.nextInt();
        if (N <= 3) {System.out.println(arr[N-1]);}
        else {System.out.println("Vacancy");}
    }
}